ioctl/client.go

```diff
@@ -204,3 +210,15 @@ func (c *client) WriteConfig(cfg config.Config) error {
 	}
 	return nil
 }
+
+func (c *client) PrintError(err error) error {
+	return output.PrintError(err)
```

Comment:
According to https://github.com/huof6829/iotex-core/blob/2e51464f4170129c7f1c7d478d63584798cc68b1/ioctl/output/format.go#L213, if Format is not empty, no error will be returned.
Thus,
```
    if err := output.PrintError(err); err != nil {
        fmt.Println(err)
    }
```
no return is needed for this function.

Reply:
yes, it always return nil

Reply:
in ioctl/cmd  the errors is printed by cobra   https://github.com/spf13/cobra/blob/v1.3.0/command.goL986  after execute RunE.   in newcmd  i add our own PrintError in ioctl.Client.   i think  after change to newcmd, we can move PintError to https://github.com/huof6829/iotex-core/blob/2e51464f4170129c7f1c7d478d63584798cc68b1/ioctl/output/format.go#L213 ,  this may be clearly